frontend/src/pages/LandingPage.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import Welcome from '../components/Welcome';
+import TopNavbar from '../components/Navbar'
+import { Router} from 'react-router-dom';
+
+class LandingPage extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+    render() {
```

Comment:
this is 4 spaces, it should be 2 spaces, and the lines below also need to be indented correctly.